kratos/includes/node.h

```diff
@@ -333,12 +333,12 @@ class Node : public Point,  public IndexedObject, public Flags
 #endif
     }
 
-    typename Node<TDimension>::Pointer Clone()
+    Pointer Clone()
     {
-        Node<3>::Pointer p_new_node = Kratos::make_shared<Node<3> >( this->Id(), (*this)[0], (*this)[1], (*this)[2]);

```

Comment:
this was intentionally 3 and not TDimension, since nodes are 3d even when we work in 2D. This should not be changed

Reply:
Ok, removing